### DISCUSSION TITLE
Czech translate

### DISCUSSION THREAD

**Discussion by skylk** [emsesp/EMS-ESP32#2095] [General]
Hi,
I am sending a translation into Czech.

[cz.zip](https://github.com/user-attachments/files/17367922/cz.zip)

**Comment by proddy** [comment]
very nice, thanks. We'll add.

**Comment by MichaelDvP** [comment]
Here is entity translation file from the actual dev as excel sheet, makes it easier to edit.
[locale_translations.xls](https://github.com/user-attachments/files/17374280/locale_translations.xls)
I've already added a cz column.
Empty fields will show up with en text, if you're unsure about the entity-name and meaning, leave it blank.

**Reply by skylk** [reply]
ok, I've already started translating.

**Comment by proddy** [comment]
Continuing in https://github.com/emsesp/EMS-ESP32/issues/2096

**Comment by proddy** [comment]
@MichaelDvP have you started on this, or shall I add it this week? Just so we don't duplicate efforts

**Reply by MichaelDvP** [reply]
Yes, i've copied to my local dev and now building to test. I can make a PR in a few minutes.
